features/internal_destinations.feature

```diff
@@ -705,6 +705,104 @@ Feature: Internal Destinations
         """
         Then we get OK response
         When we get "/published?where=%7B%22processed_from%22%3A%22123%22%7D"
+        Then we get list with 1 items
+        When we get "/published"
+        Then we get list with 2 items
```

Comment:
why are there 2 published items created?

Reply:
because of `Internal_Destination_Auto_Publish` macro